client/src/test/java/io/pravega/client/stream/impl/ControllerImplTest.java

```diff
@@ -625,8 +626,13 @@ public void deleteScope(ScopeInfo request, StreamObserver<DeleteScopeStatus> res
                 .build()
                 .start();
         executor = Executors.newSingleThreadScheduledExecutor();
-        controllerClient = new ControllerImpl(URI.create("tcp://localhost:" + serverPort),
-                ControllerImplConfig.builder().retryAttempts(1).build(), executor, creds, testSecure, "../config/cert.pem" );
+        controllerClient = new ControllerImpl( ControllerImplConfig.builder()
+                .clientConfig(
+                        PravegaClientConfig.builder().controllerURI(URI.create("tcp://localhost:" + serverPort))
+                                           .enableTls(testSecure)
+                                           .pravegaTrustStore("../config/cert.pem")
```

Comment:
I'm wondering if we should really call this "truststore".   It's really a cert or cert chain "to be trusted".    When I hear trust store, I'm thinking of a pkcs12 or jks truststore file, containing trusted entries:  https://stackoverflow.com/questions/24555890/using-a-custom-truststore-in-java-as-well-as-the-default-one

I don't know if we should support both ways to supply certs and if not, which one makes the most sense to allow. In either case, we must make it a little bit clearer what is expected as input. Currently, I think it's a bit misleading.